packages/headless/src/api/search/search/stemming.ts

```diff
@@ -4,3 +4,12 @@
  * E.g., for the query "indexing consultant", where a result contains "Techzample is a consulting firm specializing in search indexes. It employs consultants that can analyze and improve your search index.", the `TermsToHighlight` might be `{ "indexing": ["indexes", "index"], "consultant": ["consulting", "consultants"] }`.
  */
 export type TermsToHighlight = {[originalTerm: string]: string[]};
+
+/**
+ * Phrases to highlight (with [stemming](https://docs.coveo.com/en/1576)) in the results.
+ *
+ * E.g., for the exact phrase "indexing consultant", where a result contains "Techzample is a consulting firm specializing in search indexes. It employs consultants that can analyze and improve your search index.", the `PhrasesToHighlight` might be `{ "indexing consultant": {"indexing": ["indexes", "index"], "consultant": ["consulting", "consultants"]}}`.
+ */
+export declare type PhrasesToHighlight = {
+  [originalPhrase: string]: TermsToHighlight;
```

Comment:
Curious, why is the `originalPhrase` an array? Could there be multiple?

Reply:
Yes, you could search for `"the bank" "the city"` which is 2 exact phrases